src/core/distributions/phylogenetics/tree/birthdeath/BirthDeathBurstProcess.cpp

```diff
@@ -444,7 +462,7 @@ double BirthDeathBurstProcess::lnQ(double t) const
         lnD_previous -= 2 * log(1 + B_previous + exp(-A * t_b) * (1 - B_previous));
         
         double this_lnD = lnD_previous;
-        this_lnD += log(1-burst+2*burst*E_previous);
+//        this_lnD += log(1-burst+2*burst*E_previous);
```

Comment:
Are we keeping this line because we are not sure which version is right?

Reply:
New version works.